### DISCUSSION TITLE
How to use SoupbinTCP Client  with Spring boot 3?

### DISCUSSION THREAD

**Discussion by ronyitc** [paritytrading/nassau#204] [Help]
I am trying to use this awesome library with Spring Boot 3. What is the recommended way to use SoupBinTCP Client with Spring boot 3?

**Comment by jvirtanen** [comment]
Hi, @ronyitc! Unfortunately I'm not familiar with Spring Boot, but this library uses [Java NIO](https://en.wikipedia.org/wiki/Non-blocking_I/O_(Java)), so any resources on using Java NIO in Spring Boot should help.

**Comment by ronyitc** [comment]
I have already added it in Spring Boot 3. Let me check test it thoroughly. I will share my integration and findings soon. @jvirtanen  Thanks for the reply.
